openlibrary/components/MergeUI/utils.js

```diff
@@ -116,19 +116,27 @@ export function get_editions(work_key) {
     const endpoint = `${work_key}/editions.json`;
     // FIXME Fetch from prod openlibrary.org, otherwise it's outdated
     const url = location.host.endsWith('.openlibrary.org') ? `https://openlibrary.org${endpoint}` : endpoint;
-    return fetch(url).then(r => r.json());
+    return fetch(url).then(r => {
+        return (r.ok) ? r.json() : {size: 0, entries: [], error: 'Editions not retrieved'};
```

Comment:
I think it's better if these completely error. Otherwise if eg there's a temporary network glitch, and the lists.json endpoint fails, instead of actually erroring the user will be presented with inaccurate information saying that there are no lists. That's more dangerous than the UI freezing and the user hitting refresh.

I think this one's a blocker.

Reply:
For this file let's change the error handling to instead of providing empty default which could mislead users, let's `alert` that something is wrong. Non-ideal, but better than hanging, and better than showing false info.